Library/Formula/smartmontools.rb

```diff
@@ -3,8 +3,8 @@
 class Smartmontools < Formula
   desc "SMART hard drive monitoring"
   homepage 'http://sourceforge.net/apps/trac/smartmontools/'
```

Comment:
Even better! I'd suggest using the https version of that as well:
`homepage "https://www.smartmontools.org"`